### DISCUSSION TITLE
Cannot see mounted images on target computer internal SCSI bus

### DISCUSSION THREAD

**Discussion by beartrash** [akuker/RASCSI#1187] [General]
Hi all,

I've compiled PiSCSI and the web interface is running correctly, but my PowerMac 7200 cannot see any of the drive images.  Is there any additional trouble shooting?  

I've removed all other devices from the SCSI bus and terminated the PiSCSI device.  Nothing pops up in the logs that indicates an error, but I'm stumped.

**Comment by rdmark** [comment]
Following up on the discussion on Discord, were you able to get a working setup with different cables?

**Comment by beartrash** [comment]
Thanks for reaching out, I tried mounting ISO and HDA images via a new DB25 cable and had the same response.  I tried the external and internal SCSI ports on  my 7200 and my Mystic color classic and neither saw anything on the SCSI bus with SCSI Probe or a patched version of Apple HD SC 7.5.3.

I'm wondering if it's something with the GPIO on the Pi Zero W I've been using.  I'm going to pick up my Dad's Pi3B+ tomorrow and test with that.  

Other than trying to rule out faulty hardware, I'm not sure what else I can test?

**Comment by beartrash** [comment]
I reconnected the Pi Zero W to the PiSCSI and all my SCSI drives showed up on my Mystic Color Classic.  Maybe the PiSCSI wasn't properly/fully connected to the PiZero?
